### ISSUE TITLE
Mysql gateway - show tables not working

### ISSUE THREAD

**Issue by rajitha703** [implydata/plyql#34]
Hi,

Following command doesn't work on the mysql gateway : 

[92] Got SQL: SHOW FULL TABLES FROM `plyql1` LIKE '%'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Failed to resolve query with Plywood.
Query:
SHOW FULL TABLES FROM `plyql1` LIKE '%'
Message:
SQL parse error: Expected "COLUMNS" but "T" found. on 'SHOW FULL TABLES FROM `plyql1` LIKE '%''
Stack:

Can this be supported anytime soon in the new mysql gateway of Plyql? This would really be helpful.

Regards

**Comment by rajitha703** [comment]
Similar issue with something like this : 

[93] Got SQL: SHOW FULL TABLES FROM `plyq1` LIKE 'wikiticker'

**Comment by vogievetsky** [comment]
This is fixed in the latest PlyQL (0.7.14)

**Comment by rajitha703** [comment]
Hi @vogievetsky , 

Thanks much. Please help me on #28 to use these latest features of Plyql.
